src/frequenz/channels/_broadcast.py

```diff
@@ -222,11 +233,19 @@ def __init__(self, uuid: UUID, name: str, maxsize: int, chan: Broadcast[T]) -> N
                 belongs to.
         """
         self._uuid = uuid
+        """The UUID to identify the receiver in the broadcast channel's list of receivers."""
+
         self._name = name
+        """The name to identify the receiver."""
+
         self._chan = chan
+        """The broadcast channel that this receiver belongs to."""
+
         self._q: Deque[T] = deque(maxlen=maxsize)
+        """The receiver's internal message queue."""
 
         self._active = True
+        """Whether the receiver is still active."""
```

Comment:
Maybe mention what happens when it is inactive, or how/when it becomes inactive? After a quick look it seems the only way to make it inactive is by converting it into a peekable.

Reply:
done